app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/system_type.rb

```diff
@@ -0,0 +1,13 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::SystemType < ::Flavor
+  supports :provisioning do
+    if ext_management_system
+      unsupported_reason_add(:provisioning, ext_management_system.unsupported_reason(:provisioning)) unless ext_management_system.supports_provisioning?
+    else
+      unsupported_reason_add(:provisioning, _('not connected to ems'))
+    end
+  end
```

Comment:
Are flavors actually supported for provisioning?  I thought templates were and you selected flavors as part of the workflow

Reply:
Not supported. I have removed these